test/unittests/iotools/utils/test_data_handling.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from string import Template
+from typing import Union
+
+import pytest
+
+
+def test_vox_to_world_space():
+    """Test function `vox_to_world_space_method_1`."""
+
+    import numpy as np
+    from nibabel.nifti1 import Nifti1Image
+
+    from clinica.iotools.utils.data_handling import vox_to_world_space_method_1
+
+    vol = 100 + 10 * np.random.randn(5, 5, 2, 100)
```

Comment:
I would fix the random seed here, WDYT?

Reply:
I've changed it, I don't know if it is what you expected, so tell me :)